CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+* 1.4.0 January 11, 2021
```

Comment:
Suggest we drop a major version on this... if even to make it easier to remember when we cut the re-package. 

```suggestion
* 2.0.0 January 11, 2021
```

Reply:
Added major bump!